### PR TITLE
Fast restart when changing transformers tap position

### PR DESCRIPTION
**Pull request by geofjamg**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

No

**What kind of change does this PR introduce?**

Bug fix

**What is the current behavior?**

When changing a tap position of a transformer, the fast restart cache is invalidated.

**What is the new behavior (if this is a feature change)?**
Fast restart works when changing a tap position.

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [ ] No

**If yes, please check if the following requirements are fulfilled**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration steps are described in the following section

**What changes might users need to make in their application due to this PR? (migration steps)**

**Other information**:

### CODE REVIEW

File: src/main/java/com/powsybl/openloadflow/AcLoadFlowFromCache.java [hunk 1]
```diff
@@ -65,6 +65,22 @@ private void configureSwitches(LfTopoConfig topoConfig) {
                 LOGGER.warn("Actionable switch '{}' does not exist", switchId);
             }
         }
+        for (String transformerId : parametersExt.getActionableTransformersIds()) {
+            Branch<?> branch = network.getBranch(transformerId);
+            if (branch != null) {
+                topoConfig.addBranchIdWithRtcToRetain(transformerId);
+                topoConfig.addBranchIdWithPtcToRetain(transformerId);
+            } else {
+                ThreeWindingsTransformer tw3 = network.getThreeWindingsTransformer(transformerId);
```
Comment (annetill) [inline src/main/java/com/powsybl/openloadflow/AcLoadFlowFromCache.java:74]: Of course, you know that here you can optimize a little bit more and just keep the leg where we have a tap changer.

File: src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java [hunk 2]
```diff
@@ -1771,6 +1791,7 @@ public static boolean equals(LoadFlowParameters parameters1, LoadFlowParameters
                 extension1.getReportedFeatures().equals(extension2.getReportedFeatures()) &&
                 extension1.getSlackBusCountryFilter().equals(extension2.getSlackBusCountryFilter()) &&
```
Comment (annetill) [inline src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java:1792]: Map<String, Object> map = new LinkedHashMap<>(65); somewhere...

Comment (geofjamg) [reply src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java:1792]: Done

File: src/main/java/com/powsybl/openloadflow/AcLoadFlowFromCache.java (hunk 1 above)
Comment (geofjamg) [reply src/main/java/com/powsybl/openloadflow/AcLoadFlowFromCache.java:74]: Does it worth it ?

**Review by annetill — APPROVED** [review]

**Pull request merged**